stdlib/public/core/Bool.swift

```diff
@@ -87,6 +87,19 @@ public struct Bool {
   public init(_ value: Bool) {
     self = value
   }
+
+  /// Returns a random Boolean
+  ///
+  /// - Parameter generator: The random number generator to use when getting a
+  ///   random Boolean.
+  /// - Returns: A random Boolean.
+  @_inlineable
+  public static func random(
+    using generator: RandomNumberGenerator = Random.default
+  ) -> Bool {
+    let random = Int8.random(in: 0 ... 1, using: generator)
```

Comment:
Could you say why you're using this relatively complicated implementation, vs. something like `generator.next() % 2 == 0`?